src/main/scala/nl/knaw/dans/easy/license/Command.scala

```diff
@@ -15,16 +15,92 @@
  */
 package nl.knaw.dans.easy.license
 
-import org.slf4j.LoggerFactory
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, FileOutputStream, OutputStream}
+
 import nl.knaw.dans.easy.license.{CommandLineOptions => cmd}
+import org.slf4j.LoggerFactory
+import rx.lang.scala.{Observable, ObservableExtensions}
+import rx.lang.scala.schedulers.IOScheduler
+import rx.schedulers.Schedulers
+
+import scala.collection.JavaConverters._
+import scala.language.postfixOps
+
+class Command(datasetLoader: DatasetLoader,
+              placeholderMapper: PlaceholderMapper,
+              templateResolver: TemplateResolver,
+              pdfGenerator: PdfGenerator)(implicit parameters: Parameters) {
```

Comment:
That clarifies a lot! Did this in 2eff6ddf569b6578ab7359013a7659185b0092cc.

@PaulBoon Notice the API change! You have to call `LicenseCreator.apply(<parameters>).createLicenseForBusinessLayer(...)` now in the wrapper class you're making.